src/GitHub.jl

```diff
@@ -39,10 +39,15 @@ end
 
 
 function get_cdn_url_converter(mainpage)
-    commit = strip(text_only(matchall(sel".commit-tease-sha", mainpage.root)))
+    commit = match(r"\b[0-9a-f]{5,40}\b", matchall(sel".commit-tease", mainpage.root)[1].attributes["src"])
```

Comment:
break this into two lines:
```
commit_ele =  matchall(sel".commit-tease", mainpage.root)[1]
commit = match(r"\b[0-9a-f]{5,40}\b", commit_ele.attributes["src"])
```

Reply:
Also when I look at that element in the webbrowser, I don't see a `src` attribute.
What is going on with that?
(It could be Github is actually showing us different sites, I guess)

Reply:
GitHub has some sort of front-end rendering. If you go to `View Page Source (viewsource:)`, you'll able to see.